stimpacks/deploy/docker.go

```diff
@@ -0,0 +1,110 @@
+package deploy
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/mitchellh/go-homedir"
+)
+
+func (d *Deploy) startDeployContainer(environment *Environment, instance *Instance) {
```

Comment:
maybe I am missing it but is the argument `environment` used at all?

Reply:
Yeah, you're right - I had some log messages using it but must have removed them.